### DISCUSSION TITLE
[Bug]

### DISCUSSION THREAD

**Discussion by Iqballalalala** [Chia-Network/chia-blockchain#9638] [Support]
### What happened?

not syncing

### Version

1.2.11

### What platform are you using?

Windows

### What ui mode are you using?

GUI

**Comment by clablanche** [comment]
I don't know if this is the same issue, but I am syncing and when it is finished, It just said "not synced" and does not try to sync again. I have to restart nodes, then it syncing and at the end the same... I am working on version 1.2.11.
I worked for months before, but from november it does not work anymore. I even tried to reinstall all. I am using the docker image (so Linux based).
